docker-for-mac/apple-silicon.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Docker Desktop for Apple silicon
+keywords: Docker Desktop, M1, Silicon, Apple,
+title: Docker Desktop for Apple silicon
+toc_min: 2
+toc_max: 3
+redirect_from:
+- /docker-for-mac/apple-m1/
+---
+
+Welcome to Docker Desktop for Apple Silicon.
+
+## Docker Desktop 3.3.1
+
+2021-04-15
+
+Click the following link to download the GA version of Docker Desktop for Apple silicon.
+
+> [Download](https://desktop.docker.com/mac/stable/arm64/Docker.dmg)
+
+### System requirements
+
+You must install **Rosetta 2** as some binaries are still Darwin/AMD64. To install Rosetta 2 manually from the command line, run the following command:
+
+    ```
```

Comment:
Somehow the formatting doesn't work, the three backticks are visible: https://deploy-preview-12663--docsdocker.netlify.app/docker-for-mac/apple-silicon/#system-requirements

Reply:
Thanks for spotting this. I think it's the tab. Fixed this on my local branch. Will merge with a couple of other changes